src/index.html

```diff
@@ -20,8 +21,369 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="body">
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="Moyo_logo"
+          class="header__logo"
+        />
+      </a>
 
-  <body>
-    <h1>Catalog</h1>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="index.html"
```

Comment:
The `href` attribute for the navigation links points to `index.html`, which might not be correct for all categories. Consider updating these links to point to the correct pages for each category.